avidemux_fr.ts

```diff
@@ -764,8 +764,8 @@ Ignorer les informations de timing ?</translation>
     </message>
     <message>
         <location line="+23"/>
-        <source>Delete selection [DEL]</source>
```

Comment:
I'm sorry, but you cannot modify source strings. The only way to do this is to submit a PR for the file from where the string has been collected by lupdate, then ask to regenerate translation files.